examples/next-login/src/pages/home/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react'
+import LogIn from './login'
+import SignIn from './signin'
+import { SwitchButton } from '@ultraviolet/ui'
+import { css } from '@emotion/react'
+const swichButtonStyle = css`
+  margin-left: 43%;
```

Comment:
This might not be the best way to center it 😛 

<img width="2560" alt="Screenshot 2024-02-12 at 10 34 57" src="https://github.com/scaleway/ultraviolet/assets/15812968/b1c6b999-74d3-4afc-92d9-58a7cf477a84">